maintainers/maintainer-list.nix

```diff
@@ -6539,6 +6539,12 @@
     githubId = 1788628;
     name = "pandaman";
   };
+  panurg = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add panurg``.